### ISSUE TITLE
https://oniontip.com/ not reachable

### ISSUE THREAD

**Issue by Dunedan** [DonnchaC/oniontip#16]
I'm not able to reach https://oniontip.com/, neither via IPv4 nor via IPv6. Tried it from different hosts in different networks to rule out local routing problems, but always get the same problem.

**Comment by infertux** [comment]
Same here.

**Comment by github-user-1** [comment]
Same for me - is the site down?

**Comment by DonnchaC** [comment]
Unfortunately I haven't been able to maintain OnionTip recently. @twilde has set up his own instance at http://tortip.com/.

**Comment by BoBeR182** [comment]
Now reachable but SSL error.

**Comment by twilde** [comment]
@BoBeR182 are you perhaps using an older browser/OS? I'm using LetsEncrypt certs and SSL test indicates it's good for all modern browser/OS combinations: https://www.ssllabs.com/ssltest/analyze.html?d=tortip.com

**Comment by BoBeR182** [comment]
I meant https://oniontip.org https://www.ssllabs.com/ssltest/analyze.html?d=oniontip.org&latest

**Comment by ghost** [comment]
Is anyone hosting this now? oniontip.org still has an expired SSL certificate and the data is from 2016.

**Comment by twilde** [comment]
My instance at https://tortip.com/ is (back) up.

**Comment by ghost** [comment]
@twilde thank you! I've updated a blog post to mention it: http://go-beyond.org/post/autonomous-crowd-funded-tor-relays/

**Comment by ghost** [comment]
@twilde Looks like it's down?

**Comment by twilde** [comment]
@teran-mckinney yeah, sorry, on a box on my home network that has been a bit flaky recently, back up now. It gets so little use I have a hard time justifying making it more reliable. Which is probably a self-fulfilling prophecy...

**Comment by Bendr0id** [comment]
Is there any interest in relaunching this service? I could host it on one of my relays.

**Comment by BoBeR182** [comment]
Please keep it running. It has been used in the past.

**Comment by 5moufl** [comment]
does someone have an instance running? both oniontip and tortip seem to be down.

**Comment by theblazehen** [comment]
@5moufl I've been considering launching an instance, but with a cryptocurrency with lower fees.

**Comment by Bendr0id** [comment]
Monero would be a good choice

Am 03.02.2018 15:40 schrieb theblazehen <notifications@github.com>:

@5moufl<https://github.com/5moufl> I've been considering launching an instance, but with a cryptocurrency with lower fees.

**Comment by theblazehen** [comment]
@Bendr0id I don't think so, as there would be no way for me to prove that I'm not stealing any of the tips. I was considering supporting multiple cryptocurrencies, if people list eg LTC, BCH and ETH in their node settings, then they get a share of all LTC, BCH and ETH donations. What's your opinion on that?

I haven't done anything like this before, but possibly it could be implemented as a smart contract?

**Comment by Bendr0id** [comment]
Didn't thought about that, you're right it makes sense to not use an untraceable coin for this scenario. LTC isn't a bad choice.

Am 05.02.2018 07:58 schrieb theblazehen <notifications@github.com>:

@Bendr0id<https://github.com/bendr0id> I don't think so, as there would be no way for me to prove that I'm not stealing any of the tips. I was considering supporting multiple cryptocurrencies, if people list eg LTC, BCH and ETH in their node settings, then they get a share of all LTC, BCH and ETH donations. What's your opinion on that?

I haven't done anything like this before, but possibly it could be implemented as a smart contract?

**Comment by theblazehen** [comment]
@twilde Seeing as your instance is down, would you consider transferring the tortip.com domain to me, once I have something working?

**Comment by theblazehen** [comment]
@Bendr0id Alright, so I'm going with ETH, BCH, LTC, and DOGE. I think that would be enough to cover most people's preferences

**Comment by twilde** [comment]
@theblazehen the domain actually expired in January and is in the pendingDelete redemption period now, the best bet would be to try to pick it up after it fully drops (I think within a month though I haven't looked at those policies recently). I'm not sure it's in a state I can actually pull it back from at this point.

**Comment by BoBeR182** [comment]
So to make the service work, tor node owners would have to generate and display an address for each currency they want to support?

**Comment by theblazehen** [comment]
@BoBeR182 Yes. You thinking that just sticking to one (LTC?) would be better? Or converting donations to BCH or LTC before sending them?

**Comment by BoBeR182** [comment]
Would sticking to BTC and using segwit addresses not solve the problem of high transaction fees. Or to delay all transactions by 24h and send them out in bundles daily.

**Comment by theblazehen** [comment]
@BoBeR182 I haven't looked at the transaction fees for segwit, I'll look into that. And from what I remember, there weren't many donations that happened. And people sending BTC to this service would still need to pay fees, batching the transactions would only save on fees when this service sends it out to relay operators.

It may be possible to generate a transaction with multiple outputs, and then have people sign it and send it directly, rather then sending it to tortip and then sending it out again. Might be a little more complicated, but I think most people would manage, and it would remove the need for any trust in tortip

**Comment by BoBeR182** [comment]
You could probably code the transaction to have multiple outputs on the site QRcode, but then relay operators would need to have segwit addresses.
